<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>654</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="4">
      <widget class="QSpinBox" name="spinPotencia">
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>450</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEditNombre"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Elegir una</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QSlider" name="horizontalVagones">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="alas">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Alas</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Numero de vagones</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Elegir una o varias</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="reparacionPinchazos">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Kit de reparación de pinchazos</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="reactores">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reactores</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tipo de combustible</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Potencia Motor</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QSpinBox" name="spinVagones">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QComboBox" name="color">
       <item>
        <property name="text">
         <string>Rojo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blanco</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Negro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Azul</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Amarillo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Verde</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Numero de vehiculos creados</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QCheckBox" name="locomotora">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Locomotora</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="motor">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Motor</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="pushButtonMatricula">
       <property name="text">
        <string>Generar matricula</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QLineEdit" name="lineEditVehiculos">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="lineEditMatricula">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QCheckBox" name="tren">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tren de aterrizaje</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QComboBox" name="ruedas">
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="pushButtonNewCar">
       <property name="text">
        <string>Crear vehiculo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="combustible">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Combustible</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre del vehiculo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QComboBox" name="tipoCombustible">
       <item>
        <property name="text">
         <string>Gasolina</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eléctrico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Queroseno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Híbrido</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diesel</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Numero de ruedas</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="ruedaRepuesto">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rueda de repuesto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSlider" name="horizontalPotencia">
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>450</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="vagones">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Vagones</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalPotencia</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinPotencia</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalVagones</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinVagones</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
